src/views/Admin/components/CliftonStrengthsUpload/index.js

```diff
@@ -0,0 +1,141 @@
+        Themes: [cs.Theme1, cs.Theme2, cs.Theme3, cs.Theme4, cs.Theme5],
+        Private: false,
+      };
+    });
+
+    CliftonStrengthsService.postCliftonStrengths(formattedData)
+      .then((data) => {
+        setUploadResults(
+          data.map((r, index) => (
+            <TableRow key={index} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+              <TableCell style={{ fontSize: '1rem' }}>{r.Email}</TableCell>
+              <TableCell style={{ fontSize: '1rem' }}>{r.AccessCode}</TableCell>
+              <TableCell
+                style={{
+                  backgroundColor: successResults.some((s) => s === r.UploadResult)
+                    ? gordonColors.secondary.greenShades.secondary
+                    : gordonColors.secondary.red,
+                  color: 'white',
+                  fontSize: '1rem',
+                }}
+              >
+                {r.UploadResult}
+              </TableCell>
+            </TableRow>
+          )),
+        );
+        setLoading(false);
+      })
+      .catch((error) => {
+        setUploadResults(<h1>An Error has occurred while uploading.</h1>);
+        setLoading(false);
+      });
+  };
+
+  return (
+    <>
+      <Card>
+        <CardHeader style={headerStyle} align="center" title={`Clifton Strengths`} />
+        <Grid container justify="center">
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => setIsSpreadsheetUploaderOpen(true)}
+            style={{ margin: '50px auto' }}
+          >
+            Upload Clifton Strengths
+          </Button>
+        </Grid>
+        {loading ? (
+          <>
+            <GordonLoader />
+            <div style={{ height: '10px' }}></div>
+          </>
+        ) : (
+          uploadResults && (
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell style={{ fontWeight: 'bold', fontSize: '1rem' }}>Email</TableCell>
+                  <TableCell style={{ fontWeight: 'bold', fontSize: '1rem' }}>
+                    Access Code
+                  </TableCell>
+                  <TableCell style={{ fontWeight: 'bold', fontSize: '1rem' }}>
+                    Upload Result
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>{uploadResults}</TableBody>
+            </Table>
+          )
+        )}
+      </Card>
+      <SpreadsheetUploader
+        open={isSpreadsheetUploaderOpen}
+        setOpen={setIsSpreadsheetUploaderOpen}
+        onSubmitData={handleSubmitStrengths}
+        title="Clifton Strengths Upload"
+        requiredColumns={[
+          'Email',
+          'DateCompleted',
+          'Theme1',
+          'Theme2',
+          'Theme3',
+          'Theme4',
+          'Theme5',
+          'AccessCode',
+        ]}
+        buttonName="Upload Strengths"
+        template={CliftonStrengthsUploadTemplate}
+      ></SpreadsheetUploader>
```

Comment:
This can be a self-closing tag
```suggestion
       />
```